modules/op-resolver-community-tasks-ref.adoc

```diff
@@ -0,0 +1,411 @@
+
+By default, an {pipelines-shortname} installation includes a set of community tasks that you can use in your pipelines. These tasks are located in the {pipelines-shortname} installation namespace, which is normally the `openshift-pipelines` namespace.
+
+[discrete]
+[id="op-ctaskref-argocd-task-sync-and-wait_{context}"]
+== argocd-task-sync-and-wait
+The `argocd-task-sync-and-wait` community task deploys an Argo CD application and waits for it to be healthy. To do so, it requires the address of the Argo CD server and authentication information, either a username and password or an authentication token.
+
+.Example usage of the `argocd-task-sync-and-wait` community task
+[source,yaml,subs="attributes+"]
+----
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: argocd-task-sync-and-wait
+spec:
+  tasks:
+    - name: argocd-task-sync-and-wait
+      params:
+        - name: application-name
+          value: example_app_name
+        - name: revision
+          value: HEAD
+        - name: flags
+          value: '--'
+        - name: argocd-version
+          value: v2.2.2
+      taskRef:
+        kind: Task
+        name: argocd-task-sync-and-wait
+----
+
+.Supported parameters for the `argocd-task-sync-and-wait` community task
+[cols="1,2,1",options="header"]
+|===
+| Parameter | Description | Default value
+|`application-name` |The name of the application to deploy.|
+|`revision` |The revision to deploy. | `HEAD`
+|`flags` |  | `--`
+|`argocd-version` |The version of the Argo CD. |`v2.2.2`
+|===
+
+[discrete]
+[id="op-ctaskref-helm-upgrade-from-repo_{context}"]
+== helm-upgrade-from-repo
+
+The `helm-upgrade-from-repo` community task installs and upgrades a Helm chart in your {OCP} cluster based on the given Helm repository and chart.
+
+.Example usage of the `helm-upgrade-from-repo` community task
+[source,yaml,subs="attributes+"]
+----
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: helm-upgrade-from-repo
+spec:
+  tasks:
+    - name: helm-upgrade-from-repo
+      params:
+        - name: helm_repo
+          value: example_helm_repository
+        - name: chart_name
+          value: example_chart_name
+        - name: release_version
+          value: v1.0.0
+        - name: release_name
+          value: helm-release
+        - name: release_namespace
+          value: ''
+        - name: overwrite_values
+          value: ''
+        - name: helm_image
+          value: 'docker.io/lachlanevenson/k8s-helm@sha256:5c792f29950b388de24e7448d378881f68b3df73a7b30769a6aa861061fd08ae'
+      taskRef:
+        kind: Task
+        name: helm-upgrade-from-repo
+----
+
+.Supported parameters for the `helm-upgrade-from-repo` community task
+[cols="1,2,1",options="header"]
+|===
+| Parameter | Description | Default value
+|`helm_repo` |Specify a Helm repository. |
+|`chart_name` |Specify chart name to be deployed. |
+|`release_version` |The Helm release version in semantic versioning format. |`v1.0.0`
```

Comment:
Remove "The" here and in the other parameters, I think we don't normally use an article at the start of a parameter description in the table (compare to how the non-community tasks are done)